src/objectsAdvanced/task4.js

```diff
@@ -0,0 +1,7 @@
+const obj = { name: 'Rutenis', surname: 'Turcinas', age: 33 };
+const field = 'surname';
+const value = 'lecturer';
+const fn = (obj, field, value) => ({ ...obj, surname: 'lecturer' })
```

Comment:
This function returns new object with hardcoded field surname, and it's value 'lecturer'. It's wrong solution, because the function has two parameters/arguments: field and value which are bringing the information: which field should be set and what value, instead of hardcoded way.